packages/masonry/docs/base.md

```diff
@@ -11,31 +11,54 @@ Make sure that all elements have the final sizes upon rendering. Image sizes sho
 
 This feature **does not** include a grid system - so you must use your own.
 
+You can add a masonry-identifier as a data-attribute to trigger events via the eventhub using `'${masonryIdentifier}:${method}'`
+
+Currently only the `appended` method is supported: `'${masonryIdentifier}:appended'`
+> Please note: Gaps can occure when items are lazily added. Consider this when designing the different item sizes.
```

Comment:
More information about the nature of these gaps would be helpful. Otherwise this won't be noticed until implementation.